app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -210,21 +217,15 @@ public void onChanged() {
     }
 
     private void setSearchBannerState() {
-        if (!"".equals(adapter.getSearchQuery())) {
+        if (!"".equals(adapter.getSearchQuery()) || adapter.isFilteringByCalloutResult()) {

```

Comment:
Hmm ok, in that case I'll revert this change. Doesn't make sense to add that functionality for older devices after all this time if it's never been there.